extensions/api/blocking_api.py

```diff
@@ -71,6 +109,57 @@ def do_POST(self):
 
             self.wfile.write(response.encode('utf-8'))
 
+        elif self.path == '/api/v1/character':
+            self.send_response(201)
+            self.send_header('Content-Type', 'application/json')
+            self.end_headers()
+
+            name = body['name']
+            context = body['context']
+            greeting = body['greeting']
+            example_dialogue = body['example_dialogue']
```

Comment:
The example_dialogue field is no longer used in the web UI. It is expected to be part of the context string in character yamls.